.github/workflows/cicd-main.yml

```diff
@@ -60,3 +65,49 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v4
+
+  # Create Docker image with CrossModel web application.
+  docker:
```

Comment:
That is a bit hard to test as we only trigger it on pushed to the main branch so I'll let the CI decide the final fate once we merge but overall the code looks good to me ;-)

Reply:
Yes, I tested it in feature and then moved to main. So I pretty confident it works :).